extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,11 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    const alwaysShowAttr = this.element.getAttribute('always-show');
+    /** @private @const {boolean} */
+    this.alwaysShow_ = (this.element.hasAttribute('always-show') &&
+        alwaysShowAttr != '0' && alwaysShowAttr != 'false');
```

Comment:
yeah - I am not sure to be honest. 

After writing this I looked up how `contenteditable` is treated. It seems that only no value (or empty string) or `"true"` enables it. Whereas the non-existance of the attribute or `"false"` value disables it. This is the officially documented values. 

All other values doesn't enable it. so `contenteditable="hello"` is equivalent to `contenteditable="false"`.

Should we go with this approach instead?